CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # 2.2.1-dev
```

Comment:
Are we ready to drop the `-dev` and ship this?

Reply:
Happy to if @munificent would like me to do it in this pr

Reply:
@munificent !!! 😡